bigbluebutton-html5/imports/ui/components/app/setLogout.js

```diff
@@ -0,0 +1,33 @@
+import { Meteor } from 'meteor/meteor';
+import Storage from '/imports/ui/services/storage';
+
+export function setLogOut() {

```

Comment:
This problably should be moved to the Auth service `/imports/ui/services/auth/index.js`. The idea is to have all Auth behaviors and methods on the same service.